### PR TITLE
:truck: Move to GiffgaffCommunity namespace

### PR DESCRIPTION
**Pull request by davwheat**
@imorland Please double-check that I haven't missed anything during the move.

I don't want anything left behind :P

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "imorland/nightmode",
+    "name": "GiffgaffCommunity/nightmode",
```
Comment (imorland) [inline composer.json:2]: In this context, this should all be lower case

File: LICENSE [hunk 2]
```diff
@@ -2,6 +2,7 @@ MIT License
 
 Copyright (c) 2018 FriendsOfFlarum
 Copyright (c) 2018 ReFlar
+Copyright (c) 2018 giffgaff
```
Comment (imorland) [inline LICENSE:5]: Year should be 2020 here ;)

Comment (davwheat) [reply LICENSE:5]: 🤦  whoops...

File: composer.json (hunk 1 above)
Comment (davwheat) [reply composer.json:2]: Okay!

**Comment by davwheat** [comment]
All fixed!

**Pull request merged**